src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    private static final String boundary = "____________________________________________________________";
+
+    public static void main(String[] args){
+
+        Scanner scan = new Scanner(System.in);
+        Task[] item = new Task[100];
+        int taskno = 0;
+        int index = 1;
+        boolean EndConvo = false;
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("\nHello from\n" + logo);
+
+        System.out.println(boundary+"\nHello! I'm Duke\n" + "What can I do for you?\n" +boundary);
+
+
+        while(!EndConvo){
+            if(!scan.hasNextLine()){
+                return;
+            } else{
+                String CommandTxt = scan.nextLine();
+                String[] Message = CommandTxt.split(" ", 2);
+
+                if (CommandTxt.equals("bye")) {
+                    EndConvo = true;
+                } else if (CommandTxt.equals("list")) {
+                    System.out.println(boundary + "\n");
+
+                    for (int i = 0; i < taskno; i++) {
+                        System.out.println(String.format("%d", index) + "." + item[i]);
+                        index++;
+                    }
+                    System.out.println(boundary);
+                    index = 1;
+                } else if (Message[0].equals("done")) {
```

Comment:
Hey! you can reformat each conditional to do specific method like if (Message[0].equals("done")) { done();}